### DISCUSSION TITLE
Unable to Download

### DISCUSSION THREAD

**Discussion by gazzilioneardad** [raspicamplayer/camplayer#91] [Camplayer OS]
Looks like the resource is set to private...anyone know what's going on?  Absolutely love this for viewing my security cameras...even donated some good money.

**Comment by webdatatech** [comment]
Has anyone downloaded it before?

**Comment by mzsolnay** [comment]
Add me to the list. Would like to give Camplayer a try, ready to donate if its working, but I am not a linux expert, so I would prefer going with the SD image ....

**Comment by HerdOn304** [comment]
I would like the image as well as I contributed to the cause. Thank you.

**Reply by freddiebd** [reply]
Did you ever end up finding an image for the full Camplayer OS?

**Comment by hornnebel** [comment]
I would like to test it to. Thank you

**Comment by SaifAqqad** [comment]
Same, I would love to try this out

**Comment by Chevey0** [comment]
Id love to be able to try this out please

**Comment by Surgikill** [comment]
Same here, os download is set to private.

**Comment by HerdOn304** [comment]
No, I ended up side loading an app on a fire stick to fix my problem.

**Comment by jmccreary** [comment]
was just checking into this project to see if there were any updates since version 20210427, really disappointed that the project seems to now be abandoned. It's a great system, very stable. Since the original download seems to be no longer working, i put the .IMG of the last version i have up on Google Drive

[https://drive.google.com/file/d/1HrcgccRQBTZ5mxBfEL5QTCtmVu5wKw1D/view?usp=sharing](https://drive.google.com/file/d/1HrcgccRQBTZ5mxBfEL5QTCtmVu5wKw1D/view?usp=sharing)

**Reply by gazzilioneardad** [reply]
OK, I'm an idiot....how do I use this img to write it to an SD card?

**Reply by kmhart10** [reply]
> OK, I'm an idiot....how do I use this img to write it to an SD card?

I know this was two weeks old, but if you haven't figured it out yet, I used Balena Etcher and it works great! 
balena.io/etcher

The installation instructions are still live at https://www.rpi-camplayer.com/docs/installation/stepbystep/

**Reply by CodeOxymoron** [reply]
It appears this project has become abandoned?
I have been trying to download CamPlayer OS from
https://www.rpi-camplayer.com/download/
but the link appears to be set as private and I am unable to access it.
I am interested in deploying this onto several devices where I work, but if the project is abandoned, I am not sure I want to put it into service.
Can anyone confirm or recommend an active project to replace this?
